### PR TITLE
Fix friend notification sounds

### PR DESCRIPTION
**Pull request by GrayHatter**
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/573)

### CODE REVIEW

File: src/friend.c [hunk 1]
```diff
@@ -356,8 +356,8 @@ void friend_notify_status(FRIEND *f, const uint8_t *msg, size_t msg_length, char
     notify(title, title_length, (char *)msg, msg_length, f, 0);
 
     if (f->online) {
-        postmessage_audio(UTOXAUDIO_PLAY_NOTIFICATION, NOTIFY_TONE_FRIEND_ONLINE, 0, NULL);
-    } else {
         postmessage_audio(UTOXAUDIO_PLAY_NOTIFICATION, NOTIFY_TONE_FRIEND_OFFLINE, 0, NULL);
+    } else {
+        postmessage_audio(UTOXAUDIO_PLAY_NOTIFICATION, NOTIFY_TONE_FRIEND_ONLINE, 0, NULL);
```
Comment (cebe) [inline src/friend.c]: is this the correct order? `if online => play offline, else play online`? if so, a comment should explain why.

**Comment by cebe** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/573#-:-KZi2MNvZg8SCsw5jdeL:bnfp4nl)*

**Comment by cebe** [comment]
Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---

*[src/friend.c, line 361 at r1](https://reviewable.io:443/reviews/utox/utox/573#-KZ_iZOAikKQ157FMINx-r1-361:-KZi2RYxSF4z7Kp9Y3ey:b32w0x) ([raw file](https://github.com/utox/utox/blob/49363cc1509926d81b99201953b8b7a1df12ddc6/src/friend.c#L361)):*
<details><summary><i>Previously, cebe (Carsten Brandt) wrote…</i></summary><blockquote>

is this the correct order? `if online => play offline, else play online`? if so, a comment should explain why.
</blockquote></details>

OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/573)*

**Comment by cebe** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/573#-:-KZi2aAr7hMtacB15x_H:bnfp4nl)*

**Comment by endoffile78** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/utox/utox/573#-:-KZi2lGmVRWU-UIas2EF:bnfp4nl)*

**Pull request merged**